src/components/Page/EditPage.story.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { text } from '@storybook/addon-knobs';
+
+import EditPage from './EditPage';
+
+const commonEditPageProps = {
+  title: 'Page Title',
+  onClose: action('onClose'),
```

Comment:
When clicked, this console error occurs
```
Uncaught TypeError: toISOString is not a function
    at String.toJSON (<anonymous>)
    at Object.<anonymous> (index.umd.js:1)
    at JSON.stringify (<anonymous>)
    at stringify (index.js:353)
    at PostmsgTransport.send (index.js:148)
    at handler (index.js:126)
    at Channel.emit (index.js:136)
    at actionHandler (action.js:47)
    at Object.ca (react-dom.production.min.js:14)
    at ja (react-dom.production.min.js:15)
```

Reply:
This isn't an issue with just this component, in fact, all actions are broken for all the stories. I made a new issue, #594